tox.ini

```diff
@@ -9,7 +9,7 @@ skipsdist = True
 
 [testenv]
 install_command =  pip install {opts} {packages}
-commands = py.test --cov=matrix {posargs} -v --ignore=tests/functional
+commands = py.test --cov=matrix {posargs} -v --ignore=tests/functional --ignore=lib --ignore=bin --ignore=local
```

Comment:
Do we need it for the functional env, since that is explicit about which tests to run?